Source/WebKit/WebProcess/Inspector/WebInspectorUI.cpp

```diff
@@ -54,17 +54,11 @@ Ref<WebInspectorUI> WebInspectorUI::create(WebPage& page)
     return adoptRef(*new WebInspectorUI(page));
 }
 
-void WebInspectorUI::enableFrontendFeatures(WebPage& page)
-{
-    page.corePage()->settings().setCSSTypedOMEnabled(true);
```

Comment:
I think we might want to keep this for CSSNestingEnabled. WI now uses CSS Nesting.

Reply:
But that was bad due to lack of caching, no? And it's also not set currently so it seems like that's better added separately? And leaving a no-op seems bad?

Reply:
I guess we can always look up git history if we want this code back for nesting